api/http/api_inventory.go

```diff
@@ -51,6 +51,7 @@ const (
 	queryParamSort           = "sort"
 	queryParamHasGroup       = "has_group"
 	queryParamValueSeparator = ":"
+	queryParamScopeSeparator = "-"
```

Comment:
can this be "/" or is that not allowed in field names?

Reply:
@alfrunes the branch I used as reference was using the dash, I like the slash more, let's give it a try indeed. Forced pushed it, should be totally transparent to API consumers.

Reply:
It works indeed, and I like the slash more.